src/RequestHandler/mqttHandler.js

```diff
@@ -1,36 +1,144 @@
 const mqtt = require('async-mqtt');
-async function  mqttHandler (serverInfo,operations) {
-  const aliveOperations = {};
-  let url;
+const randExp = require('randexp');
+function randomChannelParamNumber (min,max) {
+  return Math.floor(Math.random() * max);
+}
+
+function replaceChannelParam (urlParameters,name,value,channelName) {
+  if (urlParameters.some((item) => item === name)) {
+    const generatedString = (!!value.regex) ? randomChannelParamString(value.regex) : null;
+    const generatedNumber = (typeof value.min === 'number') && (typeof value.max === 'number') ? randomChannelParamNumber(value.min, value.max) : null;
+    let parameterValue;
+    if (generatedString  || typeof generatedNumber === 'number')
+      if (typeof generatedNumber === 'number')  parameterValue = generatedNumber;
+      else parameterValue = generatedString;
+    else
+      parameterValue = value;
+    return channelName.replace(`{${name}}` , parameterValue);
+  }
+}
 
+function runOperationForChannel (channelName,details,client,cycles,interval,aliveOperations,operatioName) {
+  let currentCycle = 0;
+
+  const channelParams = Object.assign(details);
+  const payload = (!!channelParams.payload) ? channelParams.payload : {};
+
+  if (!!payload)  delete channelParams.payload;
+  else console.log(`\nNotice: Channel ${channelName} has no payload.`);
+
+  const urlParameters = channelName.match(new RegExp(/{(.*?)}/gm)).map((item) => item.substring(1, item.length - 1));
+
+  for (const [name, value] of Object.entries(channelParams)) {
+    channelName = replaceChannelParam(urlParameters,name,value,channelName);
+  }
+
+  const actionLoop = setInterval(async () => {
+    currentCycle += 1;
+    console.log(channelName);
+    await client.publish(channelName, JSON.stringify(payload));
+    if (currentCycle === cycles)
+      clearInterval(actionLoop);
+  }, interval);
+
+  aliveOperations[String(operatioName)] = {
+    loopInstance: actionLoop
+  };
+}
+
+function randomChannelParamString (regex) {
+  return new randExp(regex).gen();
+}
+
+function getChannelParams (channel) {
+  return channel.match(new RegExp(/{(.*?)}/gm)).map((item) => item.substring(1, item.length - 1));
+}
+/* eslint-disable security/detect-object-injection */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable prefer-const */
+// eslint-disable-next-line sonarjs/cognitive-complexity
```

Comment:
What is the reason for disabling all the rules?

Reply:
i removed the rules and moved the disable prefer-const where it should be. Did not delete it cause it seems that eslint has an issue with me using shorthand syntax to declare variables to get enumerator values.
for(let [name,value] of Object.entries(channelParams)) . value is reasigned and name is not. sadly i cannot declare one to be let and other to be constant. Also eslint-disable-next-line sonarjs/cognitive-complexity cannot be removed. Me using a function as context in other to return a interface of functions binded to that context is a programming paradigm in javascript. I dont think it is classified as an error.